view.py

```diff
@@ -18,10 +18,9 @@ def view_applicant_details(self,applicant):
 
     def view_applicants_list(self,shortlist):
         print(f"List of applicants for {shortlist.role.job_title}:")
-
-        for applicant in shortlist.applicants:
-            i = shortlist.applicants.index(applicant)
-            print(f"{i+1}. {applicant.name}")
+        
+        for index, applicant in enumerate(shortlist.applicants):
+            print(f"{index+1}. {applicant.name}")
```

Comment:
You can remove the first line "List of applicants..." - no need for that. Then you only need to pass in the applicants list.